eventyay_business/apps.py

```
@@ -0,0 +1,28 @@
+from django.utils.translation import gettext_lazy
+
+from . import __version__
+
+try:
+    from eventyay.base.plugins import PluginConfig
+except ImportError:
+    raise RuntimeError("Please use eventyay 2.7 or above to run this plugin!")
```

Ruff (0.16.3)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)